### PR TITLE
Fix CompGov alert

### PR DESCRIPTION
**Pull request by wtgodbe**
Matches update from https://github.com/dotnet/aspnetcore/pull/41270

### CODE REVIEW

File: src/SignalR/clients/ts/signalr-protocol-msgpack/package.json [hunk 1]
```diff
@@ -45,7 +45,8 @@
   ],
   "dependencies": {
     "@microsoft/signalr": "link:../signalr",
-    "@msgpack/msgpack": "^2.7.0"
+    "@msgpack/msgpack": "^2.7.0",
+    "eventsource": "^2.0.2"
```
Comment (BrennanConroy) [inline src/SignalR/clients/ts/signalr-protocol-msgpack/package.json:49]: Why is this being added? msgpack has no need for an eventsource dependency

**Review by dougbu — APPROVED** [review]

File: src/SignalR/clients/ts/signalr-protocol-msgpack/package.json (hunk 1 above)
Comment (dougbu) [reply src/SignalR/clients/ts/signalr-protocol-msgpack/package.json:49]: Weird, `yarn install` did everything for me

**Review by halter73 — APPROVED** [review]

File: src/SignalR/clients/ts/signalr-protocol-msgpack/package.json (hunk 1 above)
Comment (wtgodbe) [reply src/SignalR/clients/ts/signalr-protocol-msgpack/package.json]: Updated yarn locally & it seems to be doing the right thing now

**Review by BrennanConroy — APPROVED** [review]

**Comment by wtgodbe** [comment]
Not yet: https://github.com/dotnet/aspnetcore/blob/fb8c9e7800619ad6ce75fc9591cd4b06c93e5164/src/SignalR/clients/ts/signalr-protocol-msgpack/yarn.lock#L31-L34

**Comment by BrennanConroy** [comment]
What is warning? Last I checked eventsource 1.1.2 was safe and some databases were falsely warning about it (but should be fixed by now).

**Comment by omajid** [comment]
Apologies. I see that https://github.com/dotnet/aspnetcore/pull/41270 says 1.1.1 is safe and the version resolved is actually 1.1.2, which should have all the fixes. Sorry about the noise.

**Pull request merged**